components/decimal/src/provider.rs

```diff
@@ -116,6 +118,19 @@ pub struct DecimalSymbolStrsBuilder<'data> {
     pub grouping_separator: Cow<'data, str>,
 }
 
+impl<'data> DecimalSymbolStrsBuilder<'data> {
+    /// Build a [`DecimalSymbolsStrs`]
+    pub fn build(&self) -> VarZeroCow<'static, DecimalSymbolsStrs> {
+        VarZeroCow::from_encodeable(self)
+    }
+}
+
+impl<'data> From<&'data DecimalSymbolsStrs> for DecimalSymbolStrsBuilder<'data> {
+    fn from(other: &'data DecimalSymbolsStrs) -> Self {
+        zerofrom::ZeroFrom::zero_from(other)
+    }
+}
```

Comment:
Thought: Should the macro generate this?

Reply:
Yeah!